infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -251,9 +261,11 @@ async function setHyperchainMetadata() {
             feeReceiverAddress = keyResults.feeReceiver;
         }
     } else {
+        useMatterlabsGeth = true;
```

Comment:
...feels more like "isLocalhost" to me, given that the decision is solely made based on `results.l1Chain !== BaseNetwork.LOCALHOST`.

Reply:
Renamed and ticket to deprecate this https://linear.app/matterlabs/issue/PLA-681/unintuitive-behavior-of-setup-database-url-and-eth-client-web3-url-env